db/migrate/20201006074043_add_fields_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddFieldsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :password, :string
+    add_column :users, :mobile, :string
+    add_column :users, :birthday, :date
+    add_column :users, :avatar, :string
```

Comment:
not simply add a column, but a possibility add avatar as well)
use carrierwave or chose some other

synchronize with a teammate who implements User crud

Reply:
teammate who implements User crud said he would do it